vantiq-operations/docs/eng/vantiq-install-closed-network-azure.md

```diff
@@ -1,7 +1,247 @@
 # Considerations for Closed Network Configuration (Azure)
 
-
 ## Internal Load Balancer
-For AKS, add the annotations to the Service definition.  
-If necessary, it is possible to specify the Subnet and IP with the annotation.  
-[Use an internal load balancer with Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/internal-lb)
+
+When creating a Load Balancer in a Closed Network, it is necessary to consider the followings.
+
+- The Load Balancer must be configured as an internal Load Balancer and not have a Public IP.  
+- The Load Balancer must be connected to the intended subnet and can be managed by the NSG.  
+
+For AKS, add the annotations to `nginx.controller.service` property in the `deploy.yaml`. If necessary, it is possible to specify Subnet and IP with the annotation. [Use an internal load balancer with Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/internal-lb). The following is an example of placing a Load Balancer on a subnet with a specified name.  
+
+```yaml
+nginx:
+  controller:
+    tls:
+      cert: xxx.xxx.full.crt
+      key: xxx.xxx.key
+    service:
+      annotations:
+        service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+        service.beta.kubernetes.io/azure-load-balancer-internal-subnet: "<target_subnet_name>"
+```
+
+### Configure Security Group
+Configure it to restrict access from the Internet. Also, since the LoadBalancer does not perform SNAT, it is necessary to set the same permissions on the AKS worker nodes as the NSG set on the LoadBalancer. This means that if the Load Balancer and the AKS worker nodes are on different Subnets, the same settings should be made on each Subnet.
```

Comment:
```suggestion
Configure it to restrict access from the Internet. Also, since the LoadBalancer does not perform SNAT, it is necessary to set the same permission rules on the AKS worker nodes as that set on the LoadBalancer. This means that if the Load Balancer and the AKS worker nodes are on different Subnets, the same settings should be done on each Subnet.
```